agent-install/agent-uninstall.sh

```diff
@@ -0,0 +1,260 @@
+#!/bin/bash
+
+# The script uninstalls Horizon agent on an edge node
+
+set -e
+
+CERTIFICATE_DEFAULT="agent-install.crt"
+SKIP_UNREGISTRATION=false
+
+DEPLOYMENT_NAME="agent"
+SERVICE_ACCOUNT_NAME="agent-service-account"
+CLUSTER_ROLE_BINDING_NAME="openhorizon-agent-cluster-rule"
+NAMESPACE="openhorizon-agent"
+SECRET_NAME="openhorizon-agent-secrets"
+CONFIGMAP_NAME="openhorizon-agent-config"
+PVC_NAME="openhorizon-agent-pvc"
+
+function now() {
+	echo `date '+%Y-%m-%d %H:%M:%S'`
+}
+
+# Logging
+VERB_SILENT=0
+VERB_CRITICAL=1
+VERB_ERROR=2
+VERB_WARNING=3
+VERB_INFO=4
+VERB_DEBUG=5
+
+VERBOSITY=3 # Default logging verbosity
+
+function log_notify() {
+    log $VERB_SILENT "$1"
+}
+
+function log_critical() {
+    log $VERB_CRITICAL "CRITICAL: $1"
+}
+
+function log_error() {
+    log $VERB_ERROR "ERROR: $1"
+}
+
+function log_warning() {
+    log $VERB_WARNING "WARNING: $1"
+}
+
+function log_info() {
+    log $VERB_INFO "INFO: $1"
+}
+
+function log_debug() {
+    log $VERB_DEBUG "DEBUG: $1"
+}
+
+function now() {
+	echo `date '+%Y-%m-%d %H:%M:%S'`
+}
+
+function log() {
+    if [ $VERBOSITY -ge $1 ]; then
+        echo `now` "$2" | fold -w80 -s
+    fi
+}
+
+function help() {
+    cat << EndOfMessage
+$(basename "$0") <options> -- uninstall agent from edge cluster
+
+    -s          - skip unregistration, default is false
+    -v          - show version
+    -l          - logging verbosity level (0-5, 5 is verbose)
+    -u          - exchange user authorization credentials
+    -d          - delete node from exchange
```

Comment:
change:
"-u    - management hub user authorization credentials"
and
"-d    - delete node from the management hub"